digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfigBuilder.java

```diff
@@ -130,6 +134,26 @@ public EcsClientConfigBuilder withTaskMemory(Optional<String> taskMemory)
         return this;
     }
 
+    public EcsClientConfigBuilder withTags(Optional<Config> tags)
+    {
+        if (tags.isPresent()) {
+            final Config tagConfig = tags.get();
+            final List<Tag> tagList = new ArrayList<>();
```

Comment:
[minor] You can also use Stream API like
```
    this.tags = Optional.of(
        tagConfig.getKeys().stream()
            .map(key -> new Tag().withKey(key).withValue(tagConfig.get(key, String.class)))
            .collect(Collectors.toList()));
```

Reply:
I refactored here 90a1fff